src/applications/vaos/services/appointment/index.js

```diff
@@ -677,3 +678,113 @@ export async function cancelAppointment({ appointment }) {
     return cancelRequestedAppointment(appointment);
   }
 }
+
+/**
+ * Get scheduled appointment information needed for generating
+ * an .ics file.
+ *
+ * @export
+ * @param {Appointment} Appointment object, facility object
+ * @param {Facility} Facility object
+ * @returns An object containing appointment information.
+ */
+export function getCalendarData({ appointment, facility }) {
+  let data = {};
+  const isAtlas = appointment?.videoData.isAtlas;
+  const isHome = isVideoHome(appointment);
+  const videoKind = appointment?.videoData.kind;
+  const isVideo = appointment?.vaos.isVideo;
+  const isCommunityCare = appointment?.vaos.isCommunityCare;
+  const isPhone = isVAPhoneAppointment(appointment);
+  const isInPersonVAAppointment = !isVideo && !isCommunityCare && !isPhone;
+  const signinText = 'Sign in to VA.gov to get details about this appointment';
+
+  if (isPhone) {
+    data = {
+      summary: 'Phone appointment',
+      location: formatFacilityAddress(facility),
+      text: `A provider will call you at ${moment
+        .parseZone(appointment.start)
+        .format('h:mm a')}`,
+      phone: getFacilityPhone(facility),
+      additionalText: [signinText],
+    };
+  } else if (isInPersonVAAppointment) {
+    data = {
+      summary: `Appointment at ${facility?.name}`,
+      location: formatFacilityAddress(facility),
+      text: `You have a health care appointment at ${facility?.name}`,
+      phone: getFacilityPhone(facility),
+      additionalText: [signinText],
+    };
+  } else if (isCommunityCare) {
```

Comment:
We have a variation of CC appointments that don't have any provider info.

Here's the appointment in our local mock data:
![Screen Shot 2021-05-19 at 3 40 41 PM](https://user-images.githubusercontent.com/634932/118875152-b3353000-b8b9-11eb-906b-8660833b7a7b.png)


Here's how it renders for me:
![Screen Shot 2021-05-19 at 3 40 07 PM](https://user-images.githubusercontent.com/634932/118875080-9f89c980-b8b9-11eb-8199-7a9980be5568.png)

Reply:
From Peter,

Peter Russo  5:04 PM
How does this look? I figure we'll just leave the "location" blank:
TITLE: Community care appointment
DESCRIPTION:
This appointment is scheduled with a community care provider. Please don't go to your local VA health facility.
[Sign in to VA.gov]{link to appt details page} to get details about this appointment

https://dsva.slack.com/archives/D01E9JGTZ0X/p1621461855002400

Reply:
@vbahinwillit Looks much better, but still one undefined:

![Screen Shot 2021-05-24 at 1 46 48 PM](https://user-images.githubusercontent.com/634932/119387254-ad16c900-bc96-11eb-9994-d5914211bb96.png)